### ISSUE TITLE
Cannot set domestic water target from an automation

### ISSUE THREAD

**Issue by augi** [BenPru/luxtronik#111]
I have automation for a long time, and I've just discovered that it looks like this in UI:
![image](https://github.com/BenPru/luxtronik/assets/273052/f02ef24c-2970-4482-a780-6ff19f54934e)
```yaml
device_id: 7957ce07ce81322fa2a1b9b9fb2d4389
domain: number
entity_id: number.luxtronik2_domestic_water_target_temperature
type: set_value
value: 48
```

If I want to choose and action on the `Domestic water` entity, then only these four actions are available.
![image](https://github.com/BenPru/luxtronik/assets/273052/9a1b69ab-b0a4-4937-8891-111ce22afc60)

I'm running `2023.04.30` version. I tried to revert to `2023.03.24`, but no change.

I can see the following errors in log, maybe they are related:
```
Ignoring invalid device info: Invalid device info {'configuration_url': 'http://192.168.68.108/', 'default_manufacturer': 'Alpha Innotec', 'default_model': '', 'default_name': 'Heatpump', 'identifiers': {('luxtronik2', '300910_041_heatpump')}, 'manufacturer': 'Novelan', 'model': 'LD9', 'name': 'Heatpump 300910-041', 'suggested_area': 'Utility room', 'sw_version': 'V2.88.2'} for 'luxtronik2' config entry: device info needs to either describe a device, link to existing device or provide extra information.
Ignoring invalid device info: Invalid device info {'configuration_url': 'https://www.heatpump24.com/DownloadArea.php?layout=2', 'default_name': 'Heating', 'identifiers': {('luxtronik2', '300910_041_heating')}, 'manufacturer': 'Novelan', 'model': 'LD9', 'name': 'Heating'} for 'luxtronik2' config entry: device info needs to either describe a device, link to existing device or provide extra information.
Ignoring invalid device info: Invalid device info {'configuration_url': 'https://www.heatpump24.com/', 'default_name': 'Domestic water', 'identifiers': {('luxtronik2', '300910_041_domestic_water')}, 'manufacturer': 'Novelan', 'model': 'LD9', 'name': 'Domestic water'} for 'luxtronik2' config entry: device info needs to either describe a device, link to existing device or provide extra information.
```

**Comment by augi** [comment]
According to the error messages, it looks like they come from this code: https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/device_registry.py#L141

So it looks like more information is required for a device.

**Comment by rhammen** [comment]
#114 has fixed the "invalid device info" errors.
We are now waiting for BenPru to create a new release, so users can get the fix easily via HACS .
Until that time you could fix the problem by manually updating the your local __init__.py with [__init__.py](https://github.com/BenPru/luxtronik/blob/main/custom_components/luxtronik/__init__.py)

**Comment by BenPru** [comment]
Should be fixed with the pre-release [2023.9.10](https://github.com/BenPru/luxtronik/releases/tag/2023.9.10)
Thanks four your support.

**Comment by augi** [comment]
Can confirm it is fixed, thanks! 👍
